packages/ui/components/ui/conditional-wrap.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from 'react';
+
+/**
+ * Little utility component to optionally wrap a React component with another
+ * React component, depending on a condition.
+ *
+ * @example
+ * ```tsx
+ * <ConditionalWrap
+ *   condition={shouldUseTooltip}
+ *   wrap={(children) => (
+ *     <Tooltip>
+ *       <TooltipTrigger>{children}</TooltipTrigger>
+ *       <TooltipContent>Here is the tooltip text.</TooltipContent>
+ *     </Tooltip>
+ *   )}
+ * >
+ *   Here is the content that may or may not need a tooltip.
+ * </ConditionalWrap>
+ * ```
+ *
+ * @see https://stackoverflow.com/a/56870316/974981
+ */
+export const ConditionalWrap = ({
```

Comment:
I created this because I was going to conditionally render a tooltip around the value view. But per Henry's comment [here](https://discord.com/channels/824484045370818580/1044457038886998137/1220609960828731472), we're only going to show equivalent values in circumstances where the parent component knows which values it's interested in and renders those specifically. So this component isn't needed for this PR anymore, but I figured I'd leave it in anyway in case we need it in the future. Thoughts?

Reply:
Sure, think we can keep this